README.md

```diff
@@ -181,23 +182,44 @@ topdir/
     AFTER/     <== root module
 ```
 
+### Collect information
+
+If this is a terravalet move-after (the default):
+
 ```
-$ terraform -chdir=BEFORE state pull > BEFORE.tfstate
-$ terraform -chdir=AFTER  state pull > AFTER.tfstate
+$ terraform -chdir=BEFORE plan -no-color > BEFORE.tfplan
+$ terraform -chdir=AFTER  plan -no-color > AFTER.tfplan
+```
+
+In this case, you can also perform a basic validation: the number of elements to add compared to the number of elements to destroy must the same:
+
+   ```
+   $ grep "Plan:" BEFORE.tfplan
+   Plan: 229 to add, 0 to change, 229 to destroy.
+   ```
+If there is a mismatch, then it means that you have missed something. Go back to editing the Terraform files.
+
+
+If this is a terravalet move-before (special case):
+
 ```
+$ terraform -chdir=BEFORE plan -no-color > BEFORE.tfplan
+```
+
+### Collect remote state
 
 ```
-$ terraform -chdir=BEFORE plan -no-color > BEFORE.tfplan   # this for any move
-$ terraform -chdir=AFTER  plan -no-color > AFTER.tfplan    # this ONLY for move-before
+$ terraform -chdir=BEFORE state pull > BEFORE.tfstate
+$ terraform -chdir=AFTER  state pull > AFTER.tfstate
 ```
 
+Make a backup of the remote states. They are needed to recover in case of errors. They must be done now.
```

Comment:
Now the rephrasing is singular so `It must be done now` (the backup).

Reply:
Fixed, thanks.